### PR TITLE
Ivysaur 104 the system must be able to get the list of classes that need to be covered from the school schedule and absences tables

### PR DESCRIPTION
**Pull request by uzagmo**

### CODE REVIEW

**Review by camfiander — COMMENTED** [review]
Can you put `getClassesToBeCovered.test.js` in `server/routes`? It should be in the same folder as the file it's testing: https://ivysaur.atlassian.net/wiki/spaces/Ivysaur/pages/9535489/Unit+Testing+Best+Practices#Write-tests-using-the-filename-pattern-*filename*.test.js

**Review by camfiander — CHANGES_REQUESTED** [review]
Left some comments around testing and performance. Overall this is great work but we can definitely improve how we're using prisma to minimize our SQL calls. 

I will try updating the `data-model` branch so it has the changes I mentioned about `ScheduledClass`

File: server/routes/getClassesToBeCovered.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const getClassesToBeCovered = require('./getClassesToBeCovered');
+
+
+test('getAbsences', async function(){
+
+    let date = new Date("2022-02-15");
+    const absences = await getClassesToBeCovered.getAbsences(date);
```
[the remark this answers is not in the document]
Comment (uzagmo) [reply server/routes/getClassesToBeCovered.test.js:8]: I don't really have the time to do this right now as I have to work on my current ticket. But I wrote my tests using the data that was created by the initialization scripts so it should be passing.

**Review by camfiander — APPROVED** [review]
Sorry for not approving this sooner, this is good to go

**Pull request merged**